soap/src/java/com/zimbra/soap/mail/message/FileSharedWithMeRequest.java

```diff
@@ -98,11 +98,18 @@ public class FileSharedWithMeRequest {
     @XmlElement(name = MailConstants.A_REMOTE_ID, required = true)
     private String ownerAccountId;
 
+    /**
+     * @zm-api-field-tag ownerAccountId
```

Comment:
field doc block does not match the field, please correct it

Reply:
updated accordingly to date field